base/src/main/java/org/ndx/aadarchi/base/providers/FromDsl.java

```diff
@@ -24,6 +23,7 @@ public class FromDsl implements ArchitectureModelProvider {
 
 	@Override
 	public Workspace describeArchitecture() {
+		System.out.println("File :" + workspace.getAbsolutePath());
```

Comment:
nope, no System.out, please prefer using a Logger ;-)

This is as simple as

@Inject Logger logger;

(where injected logger is a java.util.logging.Logger)

Reply:
No, it was only for fast debug purpose. I'll remove it tomorrow night if you don't need it.

But thank for the tips, didn't know we could inject some logger this way.